test/unit/conftest.py

```diff
@@ -90,3 +90,29 @@ def noise_free_map() -> rs.DataSet:
 @pytest.fixture
 def noisy_map() -> rs.DataSet:
     return displaced_single_atom_difference_map_coefficients(noise_sigma=0.03)
+
+
+@pytest.fixture
+def atom_and_noisy_atom() -> rs.DataSet:
```

Comment:
what is this? How could I use it in other tests? better name?

Reply:
good point, updated accordingly